content/en/docs/concepts/overview/working-with-objects/owners-dependents.md

```diff
@@ -6,9 +6,10 @@ weight: 90
 
 <!-- overview -->
 
-In Kubernetes, some objects are *owners* of other objects. For example, a
-{{<glossary_tooltip text="ReplicaSet" term_id="replica-set">}} is the owner of a set of Pods. These owned objects are *dependents*
-of their owner. 
+In Kubernetes, some {{< glossary_tooltip text="objects" term_id="object" >}} are
+*owners* of other objects. For example, a
+{{<glossary_tooltip text="ReplicaSet" term_id="replica-set">}} is the owner
+of a set of Pods. These owned objects are *dependents* of their owner.
```

Comment:
nit: again, does it help to italicize?

Reply:
We're introducing a new concept term here, so I would emphasize.